mne/gui/_file_traits.py

```diff
@@ -357,6 +371,14 @@ def _get_rpa(self):
         else:
             return np.zeros((1, 3))
 
+    @cached_property
+    def _get_eeg_points(self):
+        if not self.inst or not self.eeg_dig:
+            return np.empty((0, 3))
+        dig = np.array([d['r'][:3] for d in self.eeg_dig])
+        dig.shape = (-1, 3)
```

Comment:
Isn't `dig.shape = (-1, 3)` redundant? If the shape of `dig` is not `(-1, 3)` Something must be wrong with `self.eeg_dig`